Project.cmake

```diff
@@ -36,6 +39,9 @@ if(WIN32)
 
 	# Install binaries in the root of the install directory
 	set(INSTALL_BIN_DIR "/" CACHE PATH "Where to place binaries in the instalation")
+elseif(APPLE)
+	set(INSTALL_BIN_DIR "MacOS" CACHE PATH "")
+	set(PROJECT_BINARY_DIR ${CMAKE_INSTALL_PREFIX})
```

Comment:
> This should probably be `BUNDLE DESTINATION ${PROJECT_BINARY_DIR}`

---

> `set(PROJECT_BINARY_DIR ${CMAKE_INSTALL_PREFIX})` Seems like a real bad idea :D

Yes, that's because it should've been `BUNDLE DESTINATION ${CMAKE_INSTALL_PREFIX}` in the first place, `set(PROJECT_BINARY_DIR ${CMAKE_INSTALL_PREFIX})` is just a workaround for that.